routes/index.js

```diff
@@ -1,7 +1,9 @@
 const router = require('express').Router();
 
 const productRouter = require('./productRouter');
+const detailProductRouter = require('./detailProductRouter');
 
 router.use('/products', productRouter);
+router.use('/products', detailProductRouter);
```

Comment:
`productRouter.js` 파일에서 관리할 경우 다음과 같이 수정할 수 있을 것 같습니다~!
```suggestion
const detailProductRouter = require('./detailProductRouter');

router.use('/products', productRouter);
```